util/hosts.py

```diff
@@ -0,0 +1,25 @@
+import random
+import requests
+import os
+
+
+class Hosts(object):
```

Comment:
This is really a transformative class to help with a specific JSON scheme.  I feel like  we should be forthcoming in the class name and provide an example in the class of what we expect should the destination file change, we'll have something to compare it to.

Reply:
Hosts just seems to generic.

Reply:
As I said earlier I didn't spend a lot of time on naming these, was just trying to put together something that works. But you are right.